api/institutions/serializers.py

```diff
@@ -39,6 +45,9 @@ class InstitutionSerializer(JSONAPISerializer):
         related_view_kwargs={'institution_id': '<_id>'},
     )
 
+    def get_absolute_html_url(self, obj):
+        return urlparse.urljoin(website_settings.DOMAIN, 'institutions/{}/'.format(obj._id))
```

Comment:
Not strictly a blocker, but if you implemented the `absolute_url` property on the `Institution` model, you could rely on the base serializer's [`get_absolute_html_url`](https://github.com/CenterForOpenScience/osf.io/blob/develop/api/base/serializers.py#L1432) definition and not have to override it here.